mne/io/bti/tests/test_bti.py

```diff
@@ -162,6 +162,45 @@ def test_info_no_rename_no_reorder():
                      ['A133', 'A158', 'A44', 'A134', 'A216'])
 
 
+def test_info_no_pdf():
```

Comment:
it was not possible to add these checks to an existing test?

how long does this test takes to run?